src/db/models/projectVersionField.ts

```diff
@@ -0,0 +1,37 @@
+import * as t from 'io-ts';
+
+import { brandedType } from '../../util/io-ts';
+import { Brand } from '../../util/types';
+import { defineIDModel } from '../util/id-model';
+import { CONDITION_FIELD_ID } from './conditionField';
+import { PROJECT_VERSION_PLAN_ID } from './projectVersionPlan';
+
+export type ProjectVersionFieldId = Brand<
+  number,
+  { readonly s: unique symbol },
+  'projectVersionField.id'
+>;
+
+export const PROJECT_VERSION_FIELD_ID = brandedType<
+  number,
+  ProjectVersionFieldId
+>(t.number);
+
+export default defineIDModel({
+  tableName: 'projectVersionField',
+  fields: {
+    generated: {
+      id: { kind: 'branded-integer', brand: PROJECT_VERSION_FIELD_ID },
+    },
+    required: {
+      conditionFieldId: { kind: 'branded-integer', brand: CONDITION_FIELD_ID },
+      projectVersionPlanId: {
+        kind: 'branded-integer',
+        brand: PROJECT_VERSION_PLAN_ID,
+      },
```

Comment:
`conditionFieldId` and `projectVersionPlanId` are actually `accidentallyOptional` as they aren't defined with `NOT NULL` constraint. 

`required` is for columns with `NON NULL` and `optional` for those without such constraint. But, there are no records with actual `NULL` values for these two columns in the database, qualifying it for `accidentallyOptional`